farmerbot/parser/parser.go

```diff
@@ -94,3 +94,41 @@ func ParseEnv(content string) (network string, mnemonicOrSeed string, keyType st
 
 	return
 }
+
+// ValidateInput validates that included, excluded and priority nodes are in the farm
```

Comment:
move this to separate validator file, and the tests as well

Reply:
within the parser pkg?

Reply:
yes